sup3r/postprocessing/collection.py

```diff
@@ -466,47 +466,40 @@ def _ensure_dset_in_output(out_file, dset, data=None):
                                attrs=attrs, data=data,
                                chunks=attrs.get('chunks', None))
 
-    def collect_flist(self, file_paths, out_file, feature, sort=False,
-                      sort_key=None, max_workers=None,
-                      target_final_meta_file=None, masked_target_meta=None):
-        """Collect a dataset from a file list with data pre-init.
-
-        Collects data that can be chunked in both space and time.
+    def _collect_flist(self, feature, masked_meta, time_index, shape,
+                       file_paths, out_file, target_final_meta,
+                       masked_target_meta, max_workers=None):
+        """Collect a dataset from a file list without getting attributes first.
 
         Parameters
         ----------
+        feature : str
+            Dataset name to collect.
+        masked_meta : pd.DataFrame
```

Comment:
This method docstring is confusing. What is the difference between `masked_meta` and `masked_target_meta`? Is the file paths a subset (chunk) of the full list of file paths? Is `target_final_meta_file` really a file path? (also target_final_meta_file is misnamed).

Reply:
Masked meta is for the file list chunk and masked target meta is for the entire file list. Good catch on the misnaming, here it's not a file.